tests/components/deconz/test_sensor.py

```diff
@@ -114,6 +116,7 @@ async def test_sensors(hass):
     assert "sensor.consumption_sensor" in gateway.deconz_ids
     assert "sensor.clip_light_level_sensor" not in gateway.deconz_ids
     assert len(hass.states.async_all()) == 5
+    assert len(gateway.entities[sensor.DOMAIN]) == 5
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)